frontend/src/components/result/ResultSummary.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { getLastLogDict } from '../../utils';
+import { keyOptions } from '../../constants';
 
 
 const ResultSummary = (props) => {
   const { result } = props;
   const lastLogDict = getLastLogDict(result);
+  const logKeys = keyOptions.filter((key) => Object.keys(lastLogDict).indexOf(key) > -1);
```

Comment:
`keyOptions.filter((key) => key in lastLogDict);` ?

Reply:
I also prefer to use `in`, but other source code uses same: https://github.com/chainer/chainerui/blob/3e3cd49ddd6d9b8f77d8f42e4a9950f2dcb679a1/frontend/src/containers/PlotContainer.jsx#L141

Reply:
3. is better, I think, thank you